src/test/tools/convert_xml_to_yaml/file.yaml

```diff
@@ -0,0 +1,32 @@
+host:
+- id: server
+  process:
+  - arguments: tgen.server.graphml.xml
+    plugin: tgen
+    starttime: '1'
+- id: client
+  process:
+  - arguments: tgen.client.graphml.xml
+    plugin: tgen
+    starttime: '2'
+option:
+  stoptime: '3600'
```

Comment:
This is sort of backwards from what I would expect. The numbers are given in quotes, and the strings are not. Is that a standard YAML thing?

Reply:
Bare (unquoted) strings are allowed in YAML, but I think can't begin with digits. It looks like the conversion is forcing everything to be a string, and leaving off the quotes where they aren't needed.

I agree it'd be better for it to make the integer-valued fields integers instead of forcing them to be strings.